src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/index.html

```diff
@@ -22,7 +22,15 @@
 </head>
 
 <body>
-    <div id="app">Loading...</div>
+    <div id="app">
+        <div id="blazor-default-loading">
+            <svg>
```

Comment:
Did the `viewbox` end up being unnecessary?

Reply:
Seems like it, I tested the WebAssembly template without using `viewbox`, and the loading circle was still scaled and centered.